src/me/mrCookieSlime/Slimefun/SlimefunStartup.java

```diff
@@ -480,6 +464,10 @@ public void onDisable() {
 		}
 	}
 
+	private void createDir(String path) {
+		if (!new File(path).exists()) new File(path).mkdirs();
```

Comment:
Not to be extra picky 🤓 , but new File(path) should be stored in a variable, rather than instantiating it twice.

Reply:
Hm...
That makes sence.